test/e2e/suite/command/sign.go

```diff
@@ -0,0 +1,33 @@
+package command
+
+import (
+	. "github.com/notaryproject/notation/test/e2e/internal/notation"
+	"github.com/notaryproject/notation/test/e2e/internal/utils"
+	. "github.com/onsi/ginkgo/v2"
+)
+
+var _ = Describe("notation sign", func() {
+	It("sign in JWS signature format", func() {
+		Host(BaseOptions(), func(notation *utils.ExecOpts, artifact *Artifact, vhost *utils.VirtualHost) {
+			notation.WithDescription("sign with JWS").
```

Comment:
Shouldn't this read like `notation sign` `with jws signature format` instead of 
`notation sign` `sign in JWS signature format` `sign with JWS` ?

Also why do we need `notation.WithDescription` ?

https://onsi.github.io/ginkgo/

Reply:
Updated.